src/systems/world-update.js

```diff
@@ -0,0 +1,192 @@
+const zeroPos = new THREE.Vector3(0, 0, 0);
+const zeroQuat = new THREE.Quaternion();
+const oneScale = new THREE.Vector3(1, 1, 1);
+const identity = new THREE.Matrix4();
+identity.identity();
+
+AFRAME.registerSystem("world-update", {
+  init() {
+    this._patchRenderFunc();
+    this._patchThreeJS();
+    this.frame = 0;
+  },
+
+  _patchThreeJS: function() {
+    THREE.Object3D.prototype.getWorldPosition = function(target) {
+      if (target === undefined) {
+        console.warn("THREE.Object3D: .getWorldPosition() target is now required");
+        target = new THREE.Vector3();
+      }
+
+      this.updateMatrices();
```

Comment:
Hmm no clean way to call the original here so I would note with a comment the monkeypatched `updateMatricies` line